.coveragerc

```diff
@@ -2,6 +2,7 @@
 omit =
     *lomond/utf8validator.py
     *lomond/examples/*
+    *lomond/selectors.py
```

Comment:
Can't test all the selectors on one platform. I could mock them to death, but then what am I testing?

Turns out there is a backport of the 'selectors' module, which I may end up switching to.